### PR TITLE
[DEVINFRA-469] Reformat java using AOSP style

### PR DESCRIPTION
**Pull request by jayvdb**
Also removes duplicate imports, updates copyright year,
adds missing `package` declarations and generated note.

### CODE REVIEW

File: java/src/com/swiftnav/sbp/drivers/HexStreamConnection.java [hunk 1]
```diff
@@ -9,21 +8,15 @@
  * EITHER EXPRESSED OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE IMPLIED
  * WARRANTIES OF MERCHANTABILITY AND/OR FITNESS FOR A PARTICULAR PURPOSE.
  */
-
-/*
```
Comment (jayvdb) [inline java/src/com/swiftnav/sbp/drivers/HexStreamConnection.java:13]: this needs to be moved after the `package ..`

File: java/src/com/swiftnav/sbp/client/SBPDriver.java [hunk 2]
```diff
@@ -11,20 +10,20 @@
...
+     * link and may block if necessary. If the link is no longer available an IOException must be
+     * thrown.
      */
     byte[] read(int len) throws IOException;
-    /** Write bytes to the SBP link.
-     * This function must write all the data bytes to the link.  If this is
-     * not possible because the link is no longer available an IOException must
-     * be thrown.
+    /**
+     * Write bytes to the SBP link. This function must write all the data bytes to the link. If this
```
Comment (jayvdb) [inline java/src/com/swiftnav/sbp/client/SBPDriver.java:25]: need to insert blank line after first period

File: java/src/com/swiftnav/sbp/client/SBPDriver.java [hunk 3]
```diff
@@ -11,20 +10,20 @@
...
 import java.io.IOException;
 
 /** Abstract interface for SBP stream drivers. */
 public interface SBPDriver {
-    /** Read bytes from the SBP link .
-     * This function must read the number of bytes specified from the link
-     * and may block if necessary.  If the link is no longer available an
-     * IOException must be thrown.
+    /**
+     * Read bytes from the SBP link . This function must read the number of bytes specified from the
```
Comment (jayvdb) [inline java/src/com/swiftnav/sbp/client/SBPDriver.java:19]: need to insert blank line after first period

File: generator/sbpg/targets/resources/sbp_java.java.j2 [hunk 4]
```diff
@@ -12,6 +12,8 @@
 
 package (((module_path)));
 
+// This file was auto-generated from yaml/(((filepath))) by generate.py. Do not modify by hand!
```
Comment (jayvdb) [inline generator/sbpg/targets/resources/sbp_java.java.j2:15]: split onto two lines as that is happening anyway by the formatter

File: java/src/com/swiftnav/sbp/drivers/SBPDriverJSSC.java [hunk 5]
```diff
@@ -1,10 +1,6 @@
-/*
- * Copyright (C) 2015 Swift Navigation Inc.
+/* Copyright (C) 2015-2021 Swift Navigation Inc.
  * Contact: https://support.swiftnav.com
  *
- * Based on community contributed code:
```
Comment (jayvdb) [inline java/src/com/swiftnav/sbp/drivers/SBPDriverJSSC.java:5]: another one to move under the `package..`

**Review by woodfell — APPROVED** [review]

**Pull request merged**